src/appleseed-max-impl/appleseedrenderer/projectbuilder.cpp

```diff
@@ -1133,10 +1137,44 @@ namespace
 
         params.insert("near_z", -view_params.hither);
 
-        asf::auto_release_ptr<renderer::Camera> camera =
-            view_params.projType == PROJ_PERSPECTIVE
-                ? asr::PinholeCameraFactory::static_create("camera", params)
-                : asr::OrthographicCameraFactory::static_create("camera", params);
+        asf::auto_release_ptr<renderer::Camera> camera;
+        if (view_params.projType == PROJ_PARALLEL)
+        {
+            camera = asr::OrthographicCameraFactory::static_create("camera", params);
+        }
+        else
+        {
+        #if MAX_RELEASE == MAX_RELEASE_R16
```

Comment:
It is conventional in C/C++ to not indent `#if` / `#else` lines, I suggest following this convention.